backend/Recodo.API/Startup.cs

```diff
@@ -23,7 +24,11 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
 
-            services.AddControllers();
+            services.AddControllers()
+                .AddFluentValidation(s =>
+                {
+                    s.RegisterValidatorsFromAssemblyContaining<Startup>();

```

Comment:
And still, I think we should put validators in the Common project. To reference it now, you can just create some dummy class there